src/integrationTest/resources/envelope.json

```diff
@@ -4,18 +4,18 @@
       "case_number": "1111222233334446",
       "po_box": "SSCSPO",
       "jurisdiction": "BULKSCAN",
-      "delivery_date": "2018-06-23T00:00:00.000Z",
-      "opening_date": "2018-06-23T00:00:00.000Z",
+      "delivery_date": "2018-06-23T02:10:30.123Z",
+      "opening_date": "2018-06-23T02:10:30.123Z",
```

Comment:
why did those change? is this related to using a different class?

Reply:
it's a better value than when the time component is zero, though